### DISCUSSION TITLE
How to export an archive of all my Gists?

### DISCUSSION THREAD

**Discussion by feklee** [community/community#84931] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

When I create an [export of my account data](https://docs.github.com/en/get-started/archiving-your-github-personal-account-and-public-repositories/requesting-an-archive-of-your-personal-accounts-data), that includes all my non-Gist repos. However, I am missing the Gist repos.

Of course, I have them cloned to my disk. However, I'd like to download a single compact archive file once in a while that I can pop into my Dropbox or elsewhere, like a FAT file system, without worrying about breaking file attributes.

**Comment by ibra-kdbra** [comment]
Unfortunately, GitHub does not provide a direct way to download all of your Gist repositories as a single compact archive file. However, you can manually clone each of your Gist repositories to your local machine and then create an archive of them.

Sorry there is no other way.

**Comment by feklee** [comment]
Thanks! So I continue as I have.

To be honest, exporting the Gists is at least as important as exporting the regular Git repos. I have lots of scripts in these Gists.

**Reply by ibra-kdbra** [reply]
Yeah it's a bit ugly to be honest, hope someday they can reach for a solution.
